shopify_features/models/profiles.yaml

```diff
@@ -261,94 +303,58 @@ models:
             from: models/rsTracksUnionPages
             where:  datediff(day,date(timestamp),date('{{timestamp.Format("2006-01-02 15:04:05")}}')) <= 365 
```

Comment:
Can you modify these features too - `active_days_in_past_7_days` and `active_days_in_past_365_days` to read from the new session table? Instead of `timestamp` in count(distinct date(timestamp)), you can use count(distinct date(session_start_time)). Once done, you can also remove the rsTracksUnionPages model completely as that's not used anymore.

Reply:
I have made proper changes for - `active_days_in_past_7_days` and `active_days_in_past_365_days` and also removed `rsTracksUnionPages` model.